api/metrics-api-jaxrs/pom.xml

```diff
@@ -177,14 +176,13 @@
           <plugin>
             <groupId>com.github.kongchen</groupId>
             <artifactId>swagger-maven-plugin</artifactId>
-            <version>2.3.3</version>
             <configuration>
               <apiSources>
                 <apiSource>
                   <locations>org.hawkular.metrics.api.jaxrs</locations>
                   <apiVersion>1.0</apiVersion>
                   <basePath>http://localhost:8080/hawkular-metrics/</basePath>
-                  <outputTemplate>${basedir}/src/main/resources/rest-doc/asciidoc.mustache</outputTemplate>
+                  <outputTemplate>https://raw.githubusercontent.com/hawkular/hawkular.github.io/swagger/asciidoc.mustache</outputTemplate>
```

Comment:
On 27 Mar 2015, at 12:20, Thomas Segismont wrote:
> module
> and unzip the file before generating the doc.
>
> Everyone will have the build tools module locally because it's required
> for building the project.

Sounds good to me.